### ISSUE TITLE
Desktop went black

### ISSUE THREAD

**Issue by daleybortolon** [linuxdeepin/developer-center#4457]
### SUMMARY | 问题概要

My desktop disappeared. Cannot set wallpapers and all my files that were on the desktop don't show anymore. I can access them through the files manager though.

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

```plain
Deepin 23 beta
```

### STEPS TO REPRODUCE | 复现步骤

1-Created 3 desktops
2- Set the wallpapers to change every 10 min.
3- Set all 3 desktops with my own wallpapers.
4- Tried x11 and  Wayland
5- Turned off my laptop.
6- Next day, after turning it on, my desktop disappered and went black

Extras: Rhythmbox, Chrome, Vivaldi and many Flatpak apps

![WhatsApp Image 2023-05-22 at 07 23 02](https://github.com/linuxdeepin/developer-center/assets/50796460/aa45912f-8949-425a-99bd-a4a92632d7f2)
.

**Comment by daleybortolon** [comment]
Not sure how to edit my report, so I'll post a video of the issue here.

https://github.com/linuxdeepin/developer-center/assets/50796460/9683c830-24c6-4dd9-929d-ef3e52395798

**Comment by BLumia** [comment]
> 6- Next day, after turning it on, my desktop disappered and went black

Will it get recovered if you wait for a while? And which session (x11 or wayland) are you trying to enter at this time?

Seems you can still drag the window around so it seems dde-desktop is not launched or launched but crashed. Could you provide the output of `dde-desktop` (run from deepin-terminal) when the desktop is black?

> so I'll post a video of the issue here.

Is the video in the comment related to the original report? The desktop is not black in that video...

**Comment by daleybortolon** [comment]
No, it doesn't recover! I've tried both sessions (x11 or wayland) and it is stihl black!
Yes, the video is related (couldn't edit the original posting). That is what happens when I press super + s! The desktop shows but flickers! When I stop pressing, it blacks out again!

output of dde-desktop:
carlos@carlos-PC:~$ dde-desktop
bash: dde-desktop: command not found
carlos@carlos-PC:~$ sudo dde-desktop
Verify your face or password:
Verification successful
sudo: dde-desktop: command not found
carlos@carlos-PC:~$ run dde-desktop
bash: run: command not found
carlos@carlos-PC:~$

**Comment by BLumia** [comment]
> bash: dde-desktop: command not found

That's weird since your first screenshot shows you have `dde-file-manager` installed. Did you somehow uninstalled `dde-desktop`? Please try `sudo apt install dde-desktop` and reboot, that will very likely fixes this issue.

**Comment by daleybortolon** [comment]
It fixed it! No idea why dde-desktop was uninstalled! Here's a list of everything I've installed so far (not from the app store): flatpak (firefox, thunderbird, OBS studio, Megasync, Stremio, Inkscape, Libreoffice); overgrive, rhythmbox ,synaptic (ubuntu version), google Chrome and Vivaldi. The only thing I uninstalled was the Libreoffice that came with the distro.
Not sure wether this helps, but there you go.
Thank you very much!
